src/Extensions/Script/Serialization/JavaScriptSerializer.cs

```diff
@@ -0,0 +1,19 @@
+// MIT License.
+
+using System.Text;
+
+namespace System.Web.Script.Serialization;
+
+// TODO: We'll want to use System.Text.Json and change it to be strongly typed
```

Comment:
please create an issue for us to remember.. as you the perf benefit.

Reply:
I'm tracking them all in #9